packages/client/src/constants/leaderboard.ts

```diff
@@ -0,0 +1,4 @@
+export const teamName = 'sekta'
+export const ratingFieldName = 'time'
+export const cursor = 0
+export const limit = 50
```

Comment:
константы стоит наименовать upperCase-ом
```suggestion
export const LIMIT = 50
```
и конкретизировать, что за лимит?

Reply:
+